db/migrate/20181002123523_add_total_disk_size_to_physical_storages.rb

```diff
@@ -0,0 +1,5 @@
+class AddTotalDiskSizeToPhysicalStorages < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_storages, :total_disk_size, :string
```

Comment:
I assume this will be in bytes, correct?

Reply:
Why string?

Reply:
From the provider PR (https://github.com/ManageIQ/manageiq-providers-lenovo/pull/233/files#diff-4d59c93e67bbc3a29593871858222736R5) it looks like this would be "100 GB"

:-1: on that this should be in bytes and the parser should do the conversion

Reply:
@bdunne @agrare I don't think this should be in bytes. If this was a simple storage unit, that's fine, but a physical storage have slots for a big number of storage units and could have dozens of TB. If bytes is really needed for standard purposes, I change it, but otherwise I think is best to keep it at least in GB.

And regarding the type, I can change it to be an integer instead once we agree on a standard unit.

Reply:
And if you have a storage array that big I'll happily take a test unit for testing, just saying ;)

Reply:
@agrare My concern was to have an unreadable number due to its size. But as requested, I changed to be in bytes and be a `bigint`.

Reply:
:+1: thanks @EsdrasVP the db doesn't have to be human readable as it'll be handled by the UI.  We do the same thing with the `Disk` model.